source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/imu/imu.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+import torch
+from collections.abc import Sequence
+from typing import TYPE_CHECKING
+
+import omni.physics.tensors.impl.api as physx
+from pxr import UsdPhysics
+
+import omni.isaac.lab.sim as sim_utils
+import omni.isaac.lab.utils.math as math_utils
+from omni.isaac.lab.markers import VisualizationMarkers
+
+from ..camera.utils import convert_orientation_convention, create_rotation_matrix_from_view
```

Comment:
If these are being used outside of camera, let's move this to `utils.math` function? Don't think it is nice of us to import camera stuff inside IMU sub-module.

Reply:
done